### PR TITLE
Upgrade start bootstrap 3 and new UI theme

### PR DESCRIPTION
**Pull request by adejanovski**
Upgrades the Bootstrap admin lib we rely on from 1.0.5 to 3.3.7 and provide a theme switcher with the following bootstrap themes (thanks to the [Bootstrap live customizer](https://www.bootstrap-live-customizer.com/) ): 

Superhero:
superhero

Solarized:
solarized

Flatly:
flatly

Yeti:
yeti

and United:
united

### CODE REVIEW

**Review by michaelsembwever — CHANGES_REQUESTED** [review]
Build doesn't pass the RAT check.

https://circleci.com/gh/thelastpickle/cassandra-reaper/38620

**Review by ossarga — APPROVED** [review]
Nice work @adejanovski !! Changes look good. Have a few non-blocking comments.

File: src/ui/app/index.js [hunk 1]
```diff
@@ -39,8 +40,8 @@ jQuery(document).ready(function($){
     }
   }
 
-  let currentCluster: string = $.urlParam('currentCluster');
-  if(!currentCluster) {
+  let currentCluster = $.urlParam('currentCluster');
+  if(!currentCluster || currentCluster == "null") {
```
[the remark this answers is not in the document]
Comment (adejanovski) [reply src/ui/app/index.js]: sure thing

File: src/ui/app/style.scss [hunk 2]
```diff
@@ -171,7 +172,64 @@ td > button {
 	margin-top: 1em;
 }
 
+ul.side-nav li {
+	width: 100%;
+}
+/*
```
[the remark this answers is not in the document]
Comment (adejanovski) [reply src/ui/app/style.scss:178]: I need this for some theming stuff. Let's keep it here for now, sorry for the mess it makes.

**Pull request merged**